README.md

```diff
@@ -1,11 +1,12 @@
 # How to Code Review
 
 This is a template for practicing code reviews. Please use this template as follows:
-1. Press the `use this template` button to create a copy of this repository. 
-2. Don't forget to tick the checkbox for including all branches. 
+
+1. Press the `use this template` button to create a copy of this repository.
+2. Don't forget to tick the checkbox for including all branches.
 3. Create a Pull Request for the `feature/product-list` branch.
 4. Start your code review!
 
 ## Spot the Issues
 
-Reviewing code can be difficult. You can use the following list to help you understand what to look for when reading code:
\ No newline at end of file
+Reviewing code can be difficult. You can use the list in your handout to help you understand what to look for when reading code.
```

Comment:
Where is th elist? Does it follow or will there be a link to the a different doc?